catroweb.yaml

```diff
@@ -1032,6 +1057,19 @@ components:
       items:
         $ref: '#/components/schemas/ProjectResponse'
 
+    ProjectReportResponse:
+      type: object
+      properties:
+        user_id:
```

Comment:
why do we need a user id in this request?

project owner is available through the project

and the reporter (a user) is available through the JWT token. User must be authorized to report